pandas/tests/series/methods/test_fillna.py

```diff
@@ -671,6 +671,27 @@ def test_fillna_categorical_with_new_categories(self, fill_value, expected_outpu
         result = ser.fillna(fill_value)
         tm.assert_series_equal(result, exp)
 
+    @pytest.mark.parametrize(
+        "fill_value, expected_output",
+        [
+            ("B", ["A", "B", "B", "B", "C"]),
+            ("C", ["A", "B", "C", "C", "C"])
+        ],
+    )
+
+    def test_series_fill(fill_value, expected_output):
```

Comment:
can you rename to tests_fillna_categorical

Reply:
yaa i will rename it.